facdb_build/sql/nysed_activeinstitutions.sql

```diff
@@ -1,17 +1,19 @@
+BEGIN;
+
 ALTER TABLE nysed_activeinstitutions 
 DROP COLUMN IF EXISTS ogc_fid;
 
 ALTER TABLE nysed_nonpublicenrollment 
 DROP COLUMN IF EXISTS ogc_fid;
 
-CREATE TABLE nysed_activeinstitutions_tmp as (SELECT
+CREATE TEMP TABLE nysed_activeinstitutions_tmp as (SELECT
```

Comment:
It looks like the table `nysed_activeinstitutions_temp` gets created twice. Once here as a temporary table, and once in line 205 as a normal table. Can this happen? If so, can we name them differently? Is there any way around the switching between `nysed_activeinstitutions_temp` and `nysed_activeinstitutions` that we do in lines 228-230?

Reply:
we can, feel free to change it, they are meant to be temporary

Reply:
Does is run having two tables named the same thing if one is a temporary table and one is a normal table?

Reply:
I'll just use `DROP TABLE nysed_activeinstitutions_tmp;` to avoid the confusion